neo/test/iotest/test_neuralynxio.py

```diff
@@ -22,14 +22,27 @@
 class CommonNeuralynxIOTest(BaseTestIO, unittest.TestCase, ):
     ioclass = NeuralynxIO
     files_to_test = [
-        # 'Cheetah_v4.0.2/original_data',
+        'BML/original_data',
+        'BML_unfilledsplit/original_data',
+        'Cheetah_v1.1.0/original_data',
+        'Cheetah_v4.0.2/original_data',
         'Cheetah_v5.5.1/original_data',
         'Cheetah_v5.6.3/original_data',
         'Cheetah_v5.7.4/original_data',
-        'Pegasus_v2.1.1',
```

Comment:
Why are `Cheetah_v6.3.2/incomplete_blocks` not any more included here?

Reply:
Not intended. Put back in.